### PR TITLE
A few minor lints and docs

### PR DESCRIPTION
**Pull request by nyurik**
* Added required packages to compile on Debian-style linux
* Inlined some format args, and removed some `&` in args (they cause about 6% slowdown that compiler cannot inline)
* a few spelling mistakes

### CODE REVIEW

File: node-graph/gcore/src/vector/style.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ const OPACITY_PRECISION: usize = 3;
 
 fn format_opacity(name: &str, opacity: f32) -> String {
 	if (opacity - 1.).abs() > 10_f32.powi(-(OPACITY_PRECISION as i32)) {
-		format!(r#" {}-opacity="{:.precision$}""#, name, opacity, precision = OPACITY_PRECISION)
+		format!(r#" {name}-opacity="{opacity:.OPACITY_PRECISION$}""#)
```
Comment (Keavon) [inline node-graph/gcore/src/vector/style.rs:17]: Just commenting on this one so I can double-check it's correct.

Comment (nyurik) [reply node-graph/gcore/src/vector/style.rs:17]: This one got auto-fixed with clippy, but yes, i believe you can inline the width too (I wrote the clippy lint for it, so I would take it with a grain of salt :D )

```
cargo clippy --workspace --allow-dirty --fix --benches --tests --bins -- -A clippy::all -W clippy::uninlined_format_args
```

Comment (Keavon) [reply node-graph/gcore/src/vector/style.rs:17]: I'm just unfamiliar with that Rust string formatter syntax. But it sounds like you're the expert on that if you wrote the lint for it!

**Review by Keavon — APPROVED** [review]
Awesome, thanks for the cleanup!

**Comment by Keavon** [comment]
!build

File: node-graph/gcore/src/vector/style.rs (hunk 1 above)
Comment (nyurik) [reply node-graph/gcore/src/vector/style.rs:17]: famous last words :) I did look into it at some point, learning it in depth, but not the most fun topic to learn tbh :)

**Comment by Keavon** [comment]
Thanks for the contribution! Please let me know if you'd like any suggestions on a next task if you're interested in contributing further. Cheers!

**Pull request merged**